changelogs/fragments/ensure_discovery_unsafe.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - Wrap intepreter discovery result to prevent templating
```

Comment:
I think we do need to note that this is a security issue but if we say that it is only present in alpha1, that should set expectations around how widely it affects people.

Reply:
do we have a `security:`  header?

Reply:
```changelogs/config.yaml``` says no.  That's probably a heading we should add in the future.